rfcs/006-delia-replication.md

```diff
@@ -0,0 +1,113 @@
+# RFC-006: Automated Multi-Deal Submission in Delia for Replication
+
+**Author:** Pete — @pete-eiger  
+**Date:** 2025-02-24
+
+## Abstract
+
+This RFC proposes modifications to the Delia storage deal flow so that, instead of submitting a single deal proposal, the client automatically submits three identical deal proposals. By looping through the proposal process three times and executing the necessary RPC calls in parallel, the client not only streamlines the storage deal process but also increases data replication in the protocol. This replication is a core feature of the Polka Storage solution, ensuring that data is redundantly stored across multiple providers for improved availability, durability, and resilience in a decentralized environment. This change lays the groundwork for further enhancements such as automated deal acceptance and flexible provider selection.
+
+## Introduction
+
+Polka Storage relies on replicating data across multiple storage providers to ensure high availability and fault tolerance. Currently, Delia handles storage deals on a one-to-one basis: a client calculates a piece CID, proposes a deal, uploads a file, and then publishes the deal. For true replication, it is desirable for a client to automatically generate multiple identical deals—specifically three—so that the same piece of data is stored redundantly.
```

Comment:
> Polka Storage relies on replicating data across multiple storage providers to ensure high availability and fault tolerance.

It doesn't "rely" it hasn't been implemented

Reply:
relies on the concept, will add the future connotation to it